nightshift/marc/marc_parser.py

```diff
@@ -19,42 +19,54 @@
 
 
 class BibReader:
+    """
+    An iterator class for extracting Sierra bib data from a file of MARC21 records
+    """
+
     def __init__(
         self,
-        marc_fh: str,
+        marc_target: Union[BytesIO, BinaryIO],
         library: str,
         hide_utf8_warnings: bool = True,
     ) -> None:
         """
-        Iterator that yields bib information extracted from MARC records
+        The constructor.
         Args:
-            marc_fh:                        MARC file's path
+            marc_target:                    MARC file or file-like object
```

Comment:
I like this update as it makes this more flexible, just wondering what made you make the change?

Reply:
`paramiko` - when I saw it is possible to open a channel and stream content from sftp to a local machine without copying a file this made more sense.

Reply:
oh cool, thanks!